src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
+abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
+}
+class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts working");
```

Comment:
The message 'Truck starts working' could be more informative. For example, it could indicate what kind of work the Truck is doing.